### PR TITLE
add simple demo flow

### PR DESCRIPTION
**Pull request by zzstoatzz**
Adds a simple demo flow

### CODE REVIEW

File: flows-simple/hello.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    """
+    name = String.load(block_name)
+    print(f"Hello {name.value}!")
+
+    if "Expert" not in name.value:
+        print("You're about to become a Prefect expert!")
+        name.value = name.value + " the Prefect Expert"
+        print(f"Your name is now {name.value}!")
+        name.save("name", overwrite=True)
```
Comment (desertaxle) [inline flows-simple/hello.py:23]: wow, this is a powerful flow

**Review by desertaxle — DISMISSED** [review]

File: flows-simple/hello.py (hunk 1 above)
Comment (zzstoatzz) [reply flows-simple/hello.py:23]: instant expertise

File: flows-simple/hello.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    else:
+        print("You're already a Prefect expert!")
+        print(
+            "Check out some Prefect recipes: https://docs.prefect.io/recipes/recipes/"
+        )
```
Comment (serinamarie) [inline flows-simple/hello.py:28]:
```suggestion
        )
        
if __name__ == "__main__":
   hello(block_name="name")
```

**Review by serinamarie — APPROVED** [review]
LGTM 🙂

File: flows-simple/hello.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    """A simple flow that says hello.
+
+    Assumes a `String` block named "name" exists in the active
+    Prefect Cloud workspace.
+
+    Args:
+        block_name: The name of the block to load. Defaults to "name".
+
+    """
+    name = String.load(block_name)
```
[the remark this answers is not in the document]
Comment (zzstoatzz) [reply flows-simple/hello.py:16]: no i don't think so, I can update the flow when the replacement exists

**Review by serinamarie — APPROVED** [review]

**Comment by serinamarie** [comment]
My only qualm is that `flows-simple/` could be renamed as `flows-starter/`

**Pull request merged**